### PR TITLE
feat: Card component Added

### PR DESCRIPTION
**Pull request by krishanu2911**
Added Card component with variation
Cards with badges
Cards with dismiss
Cards with text overlay
Text only cards
Vertical/Horizontal cards
Cards with shadow

https://user-images.githubusercontent.com/75560307/152703805-c17580da-c319-4ab5-9801-59c4e72a13a0.mp4

### CODE REVIEW

File: CSS/components/card.css [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+.card-close-btn:hover{
+    background-color: var(--lightest-gray);
+}
+.text-overlap{
+    position: absolute;
+    bottom: 0;
+    width: 100%;
+    background-image: linear-gradient(to bottom, rgba(184, 193, 236,0), rgba(184, 193, 236, 1));
+}
+.flex-row{
```
Comment (dev-enforced) [inline CSS/components/card.css:62]: You can use this as a utility class in the reset file named as maple-flex which has this property  so you can pass it directly into the html hence reusing it multiple times.

File: CSS/components/card.css [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    background-color: var(--active-hover-color);
+    color: white;
+}
+.small-text{
+    font-size: 0.8rem;
+}
+.hearder-text-primary{
+    color: var(--button-text-color);
+}
+.padding-left-right-sm{
```
Comment (dev-enforced) [inline CSS/components/card.css]: You can rename it as pd-horizontal less words to denote the meaning else this one is also fine to go with.

**Review by dev-enforced — CHANGES_REQUESTED** [review]
Looks really good @krishanu2911 both the cards and the code written to produce it.Gave some suggestions and then you can merge it.

File: CSS/components/card.css (hunk 1 above)
Comment (krishanu2911) [reply CSS/components/card.css:62]: Thank you Roshan for pointing out updated changes.

File: CSS/components/card.css (hunk 2 above)
Comment (krishanu2911) [reply CSS/components/card.css]: Ya renamed the class name to pd-horizontal

**Review by dev-enforced — APPROVED** [review]

**Review by simrk — COMMENTED** [review]
Lgtm.

**Pull request merged**